src/_locales/en/messages.json

```diff
@@ -245,6 +245,15 @@
     "welcome_1b_qr_3": {
         "message": "3. click \"Setup for Mobile\""
     },
+    "welcome_1b_hint_identity": {
+        "message": "We are to restore identities of..."
```

Comment:
Maybe use words like "This backup file contains the following accounts", "And we're going to request permissions for these sites:", "Restore the backup"